runtime/vara/src/lib.rs

```diff
@@ -1091,7 +1091,7 @@ impl pallet_gear_voucher::Config for Runtime {
     type Currency = Balances;
     type PalletId = VoucherPalletId;
     type WeightInfo = weights::pallet_gear_voucher::SubstrateWeight<Runtime>;
-    type CallsDispatcher = Gear;
+    type CallsDispatcher = pallet_gear::DefaultDispatcher<Runtime>;
```

Comment:
What's the difference?

Reply:
Previously call dispatcher trait was on GearPallet, but GearPallet doesn't have access to GearVoucher fns, since its Config doesn't require `: pallet_gear_voucher: Config`, so this wrap accepts runtime with both impls, furthermore such approach allows partially implement dispatcher for some other tests avoiding bringing all of the pallets at a time